src/identity/backends/sql/local_user/list.rs

```diff
@@ -0,0 +1,110 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use sea_orm::DatabaseConnection;
+use sea_orm::entity::*;
+use sea_orm::query::*;
+use sea_orm::{Cursor, SelectModel};
+
+use super::super::local_user;
+use crate::config::Config;
+use crate::db::entity::{
+    local_user as db_local_user, password as db_password,
+    prelude::{LocalUser, User as DbUser, UserOption},
+    user as db_user,
+};
+use crate::error::DbContextExt;
+use crate::identity::backends::error::IdentityDatabaseError;
+use crate::identity::types::*;
+
+/// Prepare the paginated query for listing local users.
+fn get_list_query(
```

Comment:
I strongly discourage from adding this in this change - you make the change bigger and harder to review